tests/validators/answers/test_number_answer_validator.py

```diff
@@ -80,6 +80,107 @@ def test_are_decimal_places_valid():
     }
 
 
+def test_maximum_set_as_string():
+    answer = {
+        "id": "total-percentage",
+        "label": "Total",
+        "mandatory": False,
+        "q_code": "10002",
+        "type": "Percentage",
+        "maximum": {"value": "100"},
+    }
+
+    validator = NumberAnswerValidator(
+        answer, get_mock_schema_with_data_version("0.0.3")
+    )
+    validator.validate_min_max_is_number()
+
+    assert validator.errors[0] == {
+        "message": validator.MIN_OR_MAX_IS_NOT_NUMERIC,
+        "answer_id": "total-percentage",
+    }
+
+
+def test_minimum_set_as_string():
+    answer = {
+        "calculated": True,
+        "description": "The total percentages should be 100%",
+        "id": "total-percentage",
+        "label": "Total",
+        "mandatory": False,
+        "q_code": "10002",
+        "type": "Percentage",
+        "minimum": {"value": "0"},
+    }
+
+    validator = NumberAnswerValidator(
+        answer, get_mock_schema_with_data_version("0.0.3")
+    )
+    validator.validate_min_max_is_number()
+
+    assert validator.errors[0] == {
+        "message": validator.MIN_OR_MAX_IS_NOT_NUMERIC,
+        "answer_id": "total-percentage",
+    }
+
+
+def test_min_and_max_set_as_string():
+    answer = {
+        "calculated": True,
+        "description": "The total percentages should be 100%",
+        "id": "total-percentage",
+        "label": "Total",
+        "mandatory": False,
+        "q_code": "10002",
+        "type": "Percentage",
+        "maximum": {"value": "100"},
+        "minimum": {"value": "0"},
+    }
+
+    validator = NumberAnswerValidator(
+        answer, get_mock_schema_with_data_version("0.0.3")
+    )
+    validator.validate_min_max_is_number()
+
+    assert validator.errors[1] == {
+        "message": validator.MIN_OR_MAX_IS_NOT_NUMERIC,
+        "answer_id": "total-percentage",
+    }
+    assert len(validator.errors) == 2
+
+
+def test_min_if_set_as_float():
```

Comment:
As above, maybe `test_valid_minimum_value_is_float` for clarity, so the tests indicate if its a valid/invalid case